src/Renderer/OrientedImageMaterial.js

```diff
@@ -0,0 +1,131 @@
+            const camera = cameras[i];
+            camera.needsUpdate = true;
+            camera.textureMatrix = new THREE.Matrix4();
+            camera.textureMatrixWorldInverse = new THREE.Matrix4();
+            textureMatrix[i] = camera.textureMatrix.clone();
+            texture[i] = new THREE.Texture();
+            distortion[i] = {};
+            distortion[i].size = camera.size;
+            if (camera.distortion) {
+                distortion[i].polynom = camera.distortion.polynom;
+                distortion[i].pps = camera.distortion.pps;
+                distortion[i].l1l2 = camera.distortion.l1l2;
+            }
+            this.group.add(camera);
+        }
+
+        this.uniforms = {};
+        this.uniforms.projectiveTextureAlphaBorder = new THREE.Uniform(this.alphaBorder);
+        this.uniforms.projectiveTextureDistortion = new THREE.Uniform(distortion);
+        this.uniforms.projectiveTextureMatrix = new THREE.Uniform(textureMatrix);
+        this.uniforms.projectiveTexture = new THREE.Uniform(texture);
+
+        if (Capabilities.isLogDepthBufferSupported()) {
+            this.defines.USE_LOGDEPTHBUF = 1;
+            this.defines.USE_LOGDEPTHBUF_EXT = 1;
+        }
+
+        this.vertexShader = textureVS;
+        this.fragmentShader = ShaderUtils.unrollLoops(textureFS, this.defines);
+    }
+
+    /**
+     * Set new textures and new position/orientation of the camera set.
+     * @param {THREE.Texture} textures - Array of {@link https://threejs.org/docs/#api/en/textures/Texture|THREE.Texture}.
+     * @param {Object} feature - New position / orientation of the set of cameras
+     * @param {THREE.Vector3} feature.position - New position.
+     * @param {THREE.Quaternion} feature.quaternion - New orientation.
+     */
+    setTextures(textures, feature) {
+        if (!textures) { return; }
+        this.group.position.copy(feature.position);
+        this.group.quaternion.copy(feature.quaternion);
+        this.group.updateMatrixWorld(true); // update the matrixWorldInverse of the cameras
+
+        for (let i = 0; i < textures.length; ++i) {
+            var oldTexture = this.uniforms.projectiveTexture.value[i];
+            this.uniforms.projectiveTexture.value[i] = textures[i];
+            if (oldTexture) {
+                oldTexture.dispose();
+            }
+            this.cameras[i].needsUpdate = true;
+        }
+    }
+
+    /**
+     * Udate the uniforms using the current value of camera.matrixWorld.
+     * Need to be called when the camera of the scene has changed.
+     * @param {THREE.Camera} viewCamera - Camera of the scene.
+     */
+    updateUniforms(viewCamera) {
+        for (var i = 0; i < this.cameras.length; ++i) {
+            const camera = this.cameras[i];
+            if (camera.needsUpdate) {
+                camera.updateMatrixWorld(true);
+                camera.textureMatrix.copy(ndcToTextureMatrix).multiply(camera.projectionMatrix);
```

Comment:
you could use `multiplyMatrices` instead of `copy` and `multiply`:
```js
camera.textureMatrix.multiplyMatrices(ndcToTextureMatrix, camera.projectionMatrix);
```